scrapinghub/client/__init__.py

```diff
@@ -29,7 +29,9 @@ def request(self, *args, **kwargs):
 class ScrapinghubClient(object):
     """Main class to work with Scrapinghub API.
 
-    :param auth: Scrapinghub APIKEY or other SH auth credentials.
+    :param auth: (optional) Scrapinghub APIKEY or other SH auth credentials.
+        If not provided, it will read, respectively, from 
+        ``SH_APIKEY`` or ``SHUB_JOBAUTH`` environment variables.
```

Comment:
You could add two things here:

- the SHUB_JOBAUTH is available when executing on Scrapy Cloud
- the token contained in SHUB_JOBAUTH may not be authorized to access some endpoints

It's not permitted to schedule jobs using SHUB_JOBAUTH, for example. But I believe it's possible to read/write/remove Collections and Frontiers, for example.

Reply:
Good point, let's clarify it to avoid any confusion :+1:

Reply:
:+1: 
Is there a link/documentation mentioning which services will work with this JWT Token?
I've seen something in this issue (https://github.com/scrapinghub/python-scrapinghub/issues/103) from @victor-torres . But I could not find anything in scrapy cloud docs :(

Reply:
I just updated the docs with the info above.
Can you take another look? :smile: